src/modules/blockchainApplication/manage/hooks/useCurrentApplication.js

```diff
@@ -40,9 +41,13 @@ export function useCurrentApplication() {
         onConfirm,
         onCancel,
       });
+      // Remove toast between application switches
+      toast.dismiss();
```

Comment:
We probably want to do this onConfirm, not just when showing confirm switch modal

Reply:
Fixed